scripts/install

```diff
@@ -150,7 +150,7 @@ ynh_replace_string --match_string="__ADMIN_TOKEN__" --replace_string="$admin_tok
 ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="$config"
 ynh_replace_string --match_string="__WEBSOCKET_PORT__" --replace_string="$websocket_port" --target_file="$config"
 ynh_replace_string --match_string="__ROCKET_PORT__" --replace_string="$rocket_port" --target_file="$config"
-ynh_replace_string --match_string="__APP__" --replace_string="$rocket_port" --target_file="$config"
+ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="$config"
```

Comment:
Indeed, shitty copy/pasta

Reply:
Yes :) corrected and tested: 
1. Install
2. Go to the mails to retrieve admin token
3. Log to the admin interface
4. send an invitation
5. Go to the mail, click on the invitation link
6. create an account
7. Login as user, logout, loginback

Reply:
1. Install - OK
2. Go to the mails to retrieve admin token - OK
3. Log to the admin interface - OK
4. send an invitation - OK
5. Go to the mail, click on the invitation link - OK
6. create an account - OK
7. Login as user, logout, loginback - KO with what I log above

I'm on ARM board btw, could explain why it failed

Reply:
Seems weird, as even arm-ci is able to install bitwarden_ynh: https://ci-apps-arm.yunohost.org/ci/job/740

Reply:
Install is ok, login not --'

Reply:
When you do `journalctl -u bitwarden -fe`, you see something related to the failed login ?

Reply:
I will launch a test on AARCH64 VM, see you in 5 hours ^^

Reply:
Test is ok on a AARCH64 VM ....
If you have more logs, it would be helpfull